### PR TITLE
Fix assets paths for bootstrap JS and CSS files

### PR DESCRIPTION
**Pull request by borNfreee**
SonataAdminBundle does not contain bootstrap assets anymore.

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -104,7 +104,7 @@ public function getConfigTreeBuilder()
                     ->arrayNode('javascripts')
                         ->defaultValue(array(
                                 'bundles/sonataadmin/vendor/jquery/dist/jquery.min.js',
```
Comment (Soullivaneuh) [inline DependencyInjection/Configuration.php]: I think this could be changed too, isn't it?

Comment (borNfreee) [inline DependencyInjection/Configuration.php]: Fixed

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -96,15 +96,15 @@ public function getConfigTreeBuilder()
                 ->children()
                     ->arrayNode('stylesheets')
                         ->defaultValue(array(
-                                'bundles/sonataadmin/vendor/bootstrap/dist/css/bootstrap.min.css',
+                                'bundles/sonatacore/vendor/bootstrap/dist/css/bootstrap.min.css',
                                 'bundles/sonatapage/sonata-page.front.css',
                             ))
                         ->prototype('scalar')->end()
-                    ->end()
+ so                   ->end()
```
Comment (Soullivaneuh) [inline DependencyInjection/Configuration.php]: Hmm... I don't think this should be here. :-)

Comment (borNfreee) [inline DependencyInjection/Configuration.php]: So stupid mistake :) Fixed

**Comment by Soullivaneuh** [comment]
Thanks. Could you please now squash your commit?

**Comment by Soullivaneuh** [comment]
Before squashing, please fix the tests.

**Comment by borNfreee** [comment]
@Soullivaneuh fixed tests related to my update and squashed.

There is only one failed test, but this is related to Page::slugify() method.

**Comment by Soullivaneuh** [comment]
Thank you @borNfreee.

**Pull request merged**